docs/source/cpp/compute.rst

```diff
@@ -260,6 +260,10 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | Function name            | Arity      | Input types        | Output type         |
 +==========================+============+====================+=====================+
+| abs                      | Unary      | Numeric            | Numeric             |
```

Comment:
We also have to add compute functions manually to python api doc
https://github.com/apache/arrow/blob/master/docs/source/python/api/compute.rst